app/assets/stylesheets/components/_organization_modal.scss

```diff
@@ -0,0 +1,29 @@
+.organization-modal {
+  background-color: $white;
+  left: 0;
+  margin: $base-spacing $base-spacing*4;
+  position: absolute;
+  right: 0;
+  top: 0;
+  z-index: 2;
+
+  &:before {
+    position: fixed;
```

Comment:
Properties should be ordered background-color, bottom, content, left, opacity, position, right, top, z-index